package.json

```
@@ -3,23 +3,26 @@
   "version": "1.0.0",
```

Should change version to 1.1.0
```suggestion
  "version": "1.1.0",
```